### PR TITLE
use INFO logging unless DEBUG_LOGGING is set

### LINKED ISSUE #8: Control logging with an environment variable
The images now use the default logging settings which is DEBUG in the 2 main loggers.

Instead, it should be possible to set an environment variable which should change the log level to at leat DEBUG/TRACE/INFO/WARN. This can be handled in the first-run script.

### PR DESCRIPTION
**Pull request by anestos**
Fixes #8

### CODE REVIEW

File: first-run [hunk 1]
```diff
@@ -5,4 +5,9 @@
 
 if [ -n "$PASSWORD" -a -z "$SKIP_INSTALL" ]; then
     unattendedinstall
+fi
+
+
+if [ -z "$DEBUG_LOGGING" ]; then
+  sed -i 's/AsyncLogger name="se.curity" level="DEBUG"/AsyncLogger name="se.curity" level="INFO"/g' "$IDSVR_HOME/etc/log4j2.xml"
```
Comment (travisspencer) [inline first-run]: do the same for `io.curity` logger as well.

Comment (anestos) [reply first-run]: did the opposite for that logger, since it is INFO by default

Comment (travisspencer) [reply first-run]: Perfect

Comment (anestos) [reply first-run]: @travisspencer we have some other loggers to DEBUG by default, should we control them with the same flag? 
i.e: se.curity.identityserver.config.ConfDListener, se.curity.identityserver.config.ConfDConnection, se.curity.identityserver.config

If so, we can globally change DEBUG for INFO and visa versa in the log4j2.xml file

**Comment by anestos** [comment]
This will not be merged, Curity as of version 5.0.0 accepts an environment variable that controls the logging level

**Pull request closed without merging**